examples/pets/src/2_main.tsx

```diff
@@ -112,6 +107,16 @@ export default function App() {
     );
 }
 
+function AcceptInvite() {
+    const navigate = useNavigate();
+    useAcceptInvite({
+        invitedObjectSchema: PetPost,
+        onAccept: (petPostID) => navigate("/pet/" + petPostID),
+    });
+
+    return  <p>Accepting invite...</p>
```

Comment:
By moving `useAcceptInvite` here we trigger it only when the invitation link is active.

Before we required a reload because the hash navigation made by pasting the invite link in the URL wasn't triggering a real page refresh and `useAcceptInvite` wasn't fired until the full page reload.